app/src/main/java/com/example/pomodorify/Settings.java

```diff
@@ -183,7 +186,16 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                 setDarkThemePreferences.setDarkThemePreferences(isChecked);
 
                 if(isChecked){
-                    //darkThemeOn();
+                    // setting theme to night mode
+                    AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES);
+                }
+
+                // if the above condition turns false
```

Comment:
Remove this comment (I think the code is readable enough that it doesn't need commenting) or atleast write it in one line and thus make the if - else structure more readable.